pkg/oauth2/tokenstore/mysqltokenstore/token_store.go

```diff
@@ -37,13 +40,61 @@ type TokenStore struct {
 
 // TokenStoreItem data item
 type TokenStoreItem struct {
-	ID        int64 `gorm:"primary_key"`
-	CreatedAt time.Time
-	ExpiredAt *time.Time
-	Code      string
-	Access    string
-	Refresh   string
-	Data      TokenStoreItemData
+	ID            string `gorm:"primary_key"`
+	Scope         string
+	ScopeId       string
+	Access        string
+	SecretKey     string
```

Comment:
(`Access`, `SecretKey`), as the pair I noticed other system may call them `ID` and `Secret`?

Reply:
sometimes maybe they are duplicated?

Reply:
Yes, we use the "access key"(access key id) and "secret key"(secret access key) in AWS

Reply:
so I suggest to use `AccessKey` instead of `Access`?

Reply:
and `Scope` to `ScopeType` will more proper I think.

Reply:
Changed to AccessKey. "Scope" includes definition of "type" when referred as the whole area which it deals with or includes.